CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ## 10.1.5
 
+#### Added
+
+* Update generic errors with actual platform errors
+
```

Comment:
This changelog entry needs to be for a new version, not for the already release 10.1.5